src/core/RecyclerListView.tsx

```diff
@@ -524,7 +524,7 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
     }
 
     private _renderStackWhenReady = (stack: RenderStack): void => {
-        if (!this._initStateIfRequired(stack)) {
+        if (!this._initStateIfRequired(stack) && !this._pendingScrollToOffset) {
```

Comment:
@naqvitalha should we rely on `onScroll` to eventually call `_renderStackWhenReady` to eventually render items, or save a 'pending stack' and call this method once pending scroll is complete?

Reply:
Offline discussion - not required since pending scroll isn't initialised when page is non-scrollable.